database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -14,10 +14,8 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->increments('id');
```

Comment:
do we need an id on this? can the northstar id be the real id? and have an index?